backend/Spring Boot/springboot_starterproject/README.md

```diff
@@ -0,0 +1,29 @@
+# Spring Boot Starter Project
+
+## 📌 Introduction
+This Project contains basic template for Spring Boot.<br>

```

Comment:
Some detailed intro can be useful here. Like apart from the template what else does it includes (eg. which API have you designed).